pie/default_optuna.json

```diff
@@ -0,0 +1,39 @@
+{
+      "params": [
+          // Example for changing a category
+          // {
+          //    "path": "cemb_type",
+          //    "type": "suggest_categorical",
+          //    "args": [["cnn", "rnn"]]
+          // }
+
+         // Example for changing an INT dimension
+         // {
+         //     "path": "cemb_dim",
+         //     "type": "suggest_int",
+         //     "args": [200, 600]
+         //}
+
+        // Example for path
+        // {
+        //  "path": "task_defaults/decoder",
```

Comment:
This won't suit all needs. I'd prefer to have the same way of specifying what parameters are searched for I had in the optimize.py script (Check configs/opt.json). I find it more natural and it is more general to use the same json structure as in the original config.json file.

Reply:
I feel like it simplify the way you decide what to optimize but I get your point of view.

Reply:
Let's have a discussion about this this week. The way Optuna works, going full dict like you did in optimize might not help (I'll have to rebuild the JSON path anyway as I need to use unique variable names).

Reply:
Well, I have done it according to your wishes ;)